.github/workflows/publish_release.yml

```diff
@@ -1,12 +1,18 @@
-name: Publish Cyclus Release
+name: Publish Stable Cyclus
 
 on:
-  release:
-    types: [published, created, edited]
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
```

Comment:
repeating comment: `release = released`?

Reply:
Just realized we need to change the image tag since `github.ref_name` doesnt exist in the context of a release https://github.com/orgs/community/discussions/64528.  I will make PRs to update the other two repos

Reply:
In opening the other PRs I found a flaw(?) in the triggers for `publish_release.yml`.  Should we add a trigger so that `publish_release.yml` runs on a PR that edits `publish_release.yml`?  (and tags images accordingly)